Client/WarFare/resource.h

```diff
@@ -271,29 +271,34 @@
 #define IDS_SKILL_POINT_NOT_YET         4103
 #define IDS_SKILL_SUCCESS_HEALING_FROM  4201
 #define IDS_SKILL_SUCCESS_HEALING_TO    4202
-#define IDS_SKILL_TOOLTIP_DOUBLE        4301
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_DUAL 4302
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID1 4303
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID10 4304
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID11 4305
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID12 4306
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID13 4307
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID14 4308
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID15 4309
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID16 4310
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID17 4311
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID2 4312
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID3 4313
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID4 4314
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID5 4315
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID6 4316
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID7 4317
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID8 4318
-#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID9 4319
-#define IDS_SKILL_TOOLTIP_NO_MANA       4320
-#define IDS_SKILL_TOOLTIP_USE_ITEM_EXIST 4321
-#define IDS_SKILL_TOOLTIP_USE_ITEM_NO   4322
-#define IDS_SKILL_TOOLTIP_USE_MANA      4323
+#define IDS_SKILL_TOOLTIP_DOUBLE        4301	// Required Item : Double Hand
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_DUAL 4302	// Required Item : Dual Hand
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID1	4303	// Required Item : All weapons
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID10 4304	// Required Item : Launcher
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID11 4305	// Required Item : Staff
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID12 4306	// Required Item : Arrow
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID13 4307	// Required Item : Javelin
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID14 4308	// Required Item : Warrior Armor
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID15 4309	// Required Item : Rogue Armor
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID16 4310	// Required Item : Magician Armor
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID17 4311	// Required Item : Priest Armor
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID2 4312	// Required Item : Dagger
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID3 4313	// Required Item : Staff
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID4 4314	// Required Item : Arrow
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID5 4315	// Required Item : Javelin
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID6 4316	// Required Item : Spear
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID7 4317	// Required Item : Shield
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID8 4318	// Required Item : Bow
+#define IDS_SKILL_TOOLTIP_NEED_ITEM_ID9 4319	// Required Item : Longbow
+#define IDS_SKILL_TOOLTIP_MANA_NO	    	4320	// Doesn't use MP						
+#define IDS_SKILL_TOOLTIP_STONE_ITEM		4321	// Item consumed : %s
+#define IDS_SKILL_TOOLTIP_STONE_NO			4322	// No item consumed
```

Comment:
For reference, this one too:

It's also named such for a specific use case: the stones.
But this isn't representative of the resource itself.

Rename this to be more generic.